src/index.js

```diff
@@ -1 +1,63 @@
-console.log('%c HI', 'color: firebrick')
+document.addEventListener('DOMContentLoaded', function() {
+  const imgUrl = "https://dog.ceo/api/breeds/image/random/4";
+  const breedUrl = "https://dog.ceo/api/breeds/list/all";
+  let dogList = document.getElementById('dog-breeds')
+  let dogContainer = document.getElementById('dog-image-container')
+
+  function displayDogs() {
+    fetch(imgUrl)
+    .then(res => res.json())
+    .then(dogs => {
+      let dogPics = Object.entries(dogs.message)
+  
+      dogPics.forEach(dog => {
+        let img = document.createElement('img')
+        img.src = dog[1]
+        img.style.height = '25em'
+        img.style.width = '25em'
+
+        dogContainer.appendChild(img)
+      })
+    })
+  }
+
+function displayBreedList() {
+  fetch(breedUrl)
+  .then(res => res.json())
+  .then(dogBreed => {
+    let dogObj = dogBreed.message
+    for(const dogBreed in dogObj){
+      let li = document.createElement('li')
+      li.innerText = dogBreed
+      
+      dogList.appendChild(li)
+
+      li.addEventListener('click', () => {
+        li.style.color = 'green'
+      })
+
+        function sortBreeds() {
+          let select = document.getElementById('breed-dropdown')
+          select.addEventListener('change', () => {
+            dogList.innerText = ''
+            let letter = select.value
+
+            for(const dogBreed in dogObj) {
+              if(dogBreed[0] === letter) {
+                let li = document.createElement('li')
+                li.innerText = dogBreed
+                dogList.appendChild(li)
+                li.addEventListener('click', () => {
+                  li.style.color = 'green'
+              })
+            }
+          } 
+        })
+      }
+    } sortBreeds()
```

Comment:
This is definitely a cool way of doing this, but to make it more readable you could make the `dogObj` globally available and extract this function. That would make the code more readable and easier to follow.